crates/nu-cli/src/menus/help_completions.rs

```diff
@@ -18,7 +18,7 @@ impl NuHelpCompleter {
         //Vec<(Signature, Vec<Example>, bool, bool)> {
         let mut commands = full_commands
             .iter()
-            .filter(|(sig, _, _, _, _)| {
```

Comment:
Screaming at the comment above that was already out of date trying to explain the type.

Tuples considered harmful